pkg/proxy/userspace/proxier.go

```diff
@@ -621,18 +621,26 @@ func (proxier *Proxier) serviceChange(previous, current *v1.Service, detail stri
 	}
 }
 
+// OnServiceAdd is called whenever creation of new service object
+// is observed.
 func (proxier *Proxier) OnServiceAdd(service *v1.Service) {
 	proxier.serviceChange(nil, service, "OnServiceAdd")
 }
 
+// OnServiceUpdate is called whenever modification of an existing
+// service object is observed.
 func (proxier *Proxier) OnServiceUpdate(oldService, service *v1.Service) {
 	proxier.serviceChange(oldService, service, "OnServiceUpdate")
 }
 
+// OnServiceDelete is called whenever deletion of an existing service
+// object is observed.
 func (proxier *Proxier) OnServiceDelete(service *v1.Service) {
 	proxier.serviceChange(service, nil, "OnServiceDelete")
 }
 
+// OnServiceSynced is called once all the initial even handlers were
```

Comment:
even**T** handlers

Probably needs to be fixed wherever you copied that from too...

Reply:
Thanks @danwinship 
Updated!